test/e2e/e2e_suite_test.go

```diff
@@ -191,6 +191,13 @@ var _ = Describe("operator-sdk", func() {
 			err = tc.Make("bundle-build", "BUNDLE_IMG="+bundleImage)
 			Expect(err).NotTo(HaveOccurred())
 
+			By("running the operator bundle unsing `run bundle` command")
+			// Push the bundle image
+			err = tc.Make("docker-push", "IMG="+bundleImage)
```

Comment:
Why are you pushing the image? I don't think `run bundle` requires the image be present remotely, nor would this work.

Reply:
We do not push the images. Since the tests use Kind we load the image. See for example:

https://github.com/operator-framework/operator-sdk/blob/13198cfc2880b95ae3bf9ecec3714fd94b16b02c/test/e2e-helm/e2e_helm_suite_test.go#L115-L123